Auth0.podspec

```diff
@@ -16,11 +16,30 @@ web_auth_files = [
   'Auth0/SilentSafariViewController.swift',
   'Auth0/NativeAuth.swift',
   'Auth0/AuthProvider.swift',
-  'Auth0/CredentialsManager.swift',
```

Comment:
this are referenced in `s.osx.exclude_files`. Why did you remove them?

Reply:
Enabling it for tvOS also allows it for Mac